liquibase-standard/src/main/java/liquibase/ExtensibleObject.java

```diff
@@ -5,7 +5,7 @@
 
 /**
  * This interface defines how objects can be extended with additional attributes at runtime without subclassing and exposes the ability to query attributes without resorting to reflection.
- *
+ * <p>
```

Comment:
For those who were unaware, as I was prior to this review, the parser infers the closing `</p>`. :)